src/apiCalls.tsx

```diff
@@ -1 +1,45 @@
-export {}
+export const fetchNewsData = () => {
+  return (
+    fetch('https://api.nytimes.com/svc/topstories/v2/home.json?api-key=GKUzDD1VY9ssjZ1AGusX3ci6AeoXCaSr')
+    .then(response => checkResponse(response))
+    .then(data=> cleanNewsData(data.results))
+  )
+}
+
+const checkResponse = (response: any) => {
```

Comment:
I added a type of any to the parameter. I wonder what would be the best approach to this to avoid using any.

Reply:
Could we set the type to object? Or go even further and specify it needs to be an object that has a property with a key of 'ok'?

Reply:
So I created the type for the response object that has the following keys: 
`
interface Response {
  ok: boolean
  status: number
  json: any
}
`

I was not able to figure out what type I should assign to json(), but will continue to look into it.